compute/k8s-services/vars.tf

```diff
@@ -1273,7 +1109,7 @@ variable "kafka_exporter_deploy" {
 }
 
 variable "kafka_exporter_clusters" {
-  type        = map
+  type        = map(any)
```

Comment:
Seems unrelated. Any reason or just cause why not?

Reply:
Format and linting on save....